components/tree/tree.component.ts

```diff
@@ -77,8 +77,8 @@ const NZ_CONFIG_COMPONENT_NAME = 'tree';
 
         <div
           *ngIf="!nzVirtualHeight"
+          class="ant-tree-list-holder-inner"
           [class.ant-select-tree-list-holder-inner]="nzSelectMode"
-          [class.ant-tree-list-holder-inner]="nzSelectMode"
```

Comment:
Only needs to change `[class.ant-tree-list-holder-inner]="nzSelectMode"` to `[class.ant-tree-list-holder-inner]="!nzSelectMode"`, the className of blocknode follows `.ant-tree.ant-tree-block-node .ant-tree-list-holder-inner .ant-tree-node-content-wrapper`